File: Leetcode/Weekly-Contest-378/Check if Bitwise OR Has Trailing Zeros.cpp
#include <vector> 
 
class Solution { 
public: 
    bool hasTrailingZeros(std::vector<int>& nums) { 
        int cnt = 0; 
        for (auto x : nums) { 
            if (x % 2 == 0) { 
                cnt++; 
            } 
        } 
        if (cnt >= 2) { 
            return true; 
        } 
        return false; 
    } 
};

File: Leetcode/Weekly-Contest-378/Check if Bitwise OR Has Trailing Zeros.java
class Solution {
    public boolean hasTrailingZeros(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                int result = nums[i] | nums[j];
                if (result % 2 == 0) {
                    return true;
                } else {
                    continue;
                }
            }
        }
        return false;
    }
}
